homework_1/zad2/taxLibrary/taxLibrary.go

```diff
@@ -0,0 +1,55 @@
+package taxLibrary
+
+import (
+	"github.com/pkg/errors"
+	"math"
+)
+
+type TaxClass struct {
+	Start      int
```

Comment:
minor nitpick - there are no guarantees Start and End are integers - for example, if the legislation in Croatia states that a class should end at 100 euros (because they are using the EU regulations), the amount in HRK will almost certainly not be an integer

Reply:
thank you for the observation, I changed the program so that it can receive float values and added a test case for float values.